cpp/src/parquet/arrow/arrow_reader_writer_test.cc

```diff
@@ -661,16 +671,18 @@ class ParquetIOTestBase : public ::testing::Test {
   void RoundTripSingleColumn(
       const std::shared_ptr<Array>& values, const std::shared_ptr<Array>& expected,
       const std::shared_ptr<::parquet::ArrowWriterProperties>& arrow_properties,
```

Comment:
```suggestion
      const std::shared_ptr<::parquet::ArrowWriterProperties>& arrow_writer_properties,
```

Would be good to explicitly add `writer` to the parameter.